src/providers/types.ts

```diff
@@ -153,3 +154,22 @@ export interface ImageGenerateResponse {
   data: object[];
   provider: string;
 }
+
+/**
+ * The structure of a completion response for the 'fimComplete' function.
+ * @interface
+ */
+export interface FimCompletionResponse extends CResponse {
+  choices: FimChoice[];
+}
+
+/**
+ * The structure of a choice in a chat completion response.
+ * @interface
+ */
+export interface FimChoice {
+  index: number;
+  message: Message;
+  finish_reason: string;
+  logprobs?: object | null;
```

Comment:
Please check if the key is optional or if it can be null

Reply:
yes

Reply:
removed as it wast there in mistral response object